fe1-web/src/features/rollCall/components/AttendeeList.tsx

```diff
@@ -0,0 +1,54 @@
+import { ListItem } from '@rneui/themed';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+
+import { PoPIcon } from 'core/components';
+import { PublicKey } from 'core/objects';
+import { Color, Icon, List, Typography } from 'core/styles';
+
+const AttendeeList = ({ popTokens }: IPropTypes) => {
+  return (
+    <View style={List.container}>
+      <ListItem.Accordion
+        containerStyle={List.accordionItem}
+        content={
+          <ListItem.Content>
+            <ListItem.Title style={[Typography.base, Typography.important]}>
+              Attendees
+            </ListItem.Title>
+          </ListItem.Content>
+        }
+        isExpanded>
+        {popTokens.map((token, idx) => {
+          const listStyle = List.getListItemStyles(idx === 0, idx === popTokens.length - 1);
+
+          return (
+            <ListItem key={token.valueOf()} containerStyle={listStyle} style={listStyle}>
+              <View style={List.icon}>
+                <PoPIcon name="qrCode" color={Color.primary} size={Icon.size} />
+              </View>
+              <ListItem.Content>
+                <ListItem.Title style={Typography.base} numberOfLines={1} selectable>
+                  {token.valueOf()}
+                </ListItem.Title>
+              </ListItem.Content>
+            </ListItem>
+          );
+        })}
+      </ListItem.Accordion>
+    </View>
+  );
+};
+
+const propTypes = {
+  popTokens: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.instanceOf(PublicKey).isRequired).isRequired,
+    PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+  ]).isRequired,
```

Comment:
do you want to support:
 either (list of `PublicKey`) or (list of `string`) 
or
 list of either (`PublicKey`) or (`string`)
?

It seems you might want the latter, no?
Although ideally, again, just one type should be enough.

Reply:
I actually wanted the former here as the latter works fine but would more easily allow weird code that has an array of mixed types. But only accepting public key is even better I think :)